src/server/serverSetup.js

```diff
@@ -41,15 +45,65 @@ fluid.defaults("sjrk.storyTelling.server", {
                     secrets: "@expand:sjrk.storyTelling.server.resolveJSONFile({that}.options.secureConfig.secretsConfigPath)"
                 },
                 port: "{that}.options.globalConfig.port",
+                session: {
+                    store: "@expand:sjrk.storyTelling.server.makeMemorySessionStore()",
+                    middlewareOptions: {
+                        secret: "{server}.options.secureConfig.secrets.session"
+                    }
+                },
                 distributeOptions: {
-                    record: "@expand:kettle.resolvers.env(COUCHDB_URL)",
-                    target: "{that sjrk.storyTelling.server.dataSource.couch.core}.options.host"
+                    couchDBURL: {
+                        record: "@expand:kettle.resolvers.env(COUCHDB_URL)",
+                        target: "{that sjrk.storyTelling.server.dataSource.couch.core}.options.host"
+                    },
+                    couchDBAuthorsURL: {
+                        record: "@expand:kettle.resolvers.env(COUCHDB_URL)",
+                        target: "{that gexpressUserUtils}.options.dataSourceConfig.host"
+                    },
+                    sessionOptions: {
+                        source: "{that}.options.session",
+                        target: "{that > kettle.middlewareHolder > session}.options"
+                    }
                 },
                 components: {
+                    // user authentication
+                    expressUserUtils: {
+                        type: "fluid.express.user.utils",
+                        options: {
+                            iterations: 10,
```

Comment:
Is 10 iterations enough? I've seen 100, 1,000 or even 10,000 iterations suggested in order to defend against brute-force attacks, especially so with faster hashing algorithms. That said, if we're heading towards cloud hosting, too high a number could make hosting very expensive...

Reply:
This is the default from the underlying library actually. We could make it some other value as needed.